src/framework/piral-core/src/types/utils.ts

```diff
@@ -21,13 +23,20 @@ export interface PiralStorage {
   removeItem(name: string): void;
 }
 
+/**
+ * Can be implemented by objects that allocate huge resources. The interface provides
```

Comment:
Has nothing to do with huge resources (what is huge?), but rather with the capability of having the opposite operator besides a creation (i.e., destruction).

So reduce the comment to remark that this is implemented by functions to be used for disposal purposes.

Reply:
adapted